docs/user-guide/work-packages/share-work-packages/README.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_navigation:
+  title: Share work packages
+  priority: 963
+description: How to share work packages in OpenProject.
+keywords: work package, share, share work packages
+---
+
+# Share work packages
+
+Since the 13.1 it is possible to share work packages with users that are not members of a project or are not yet registered on your instance. In the latter case a user will need to register in order to view the work package.
+
+> **Note**: Sharing work packages with non-member is an Enterprise add-on and can only be used with [Enterprise cloud](../../../enterprise-guide/enterprise-cloud-guide/) or  [Enterprise on-premises](../../../enterprise-guide/enterprise-on-premises-guide/). An upgrade from the free community edition is easy and helps support OpenProject.
```

Comment:
Maybe we need to say somewhere also that you need to have the correct rights to share work packages.

Reply:
If the user doesn't have the correct rights they will not be able to add users in the share modal (only view)